src/modules/Layout/Table/Table.scss

```diff
@@ -0,0 +1,57 @@
+@import 'index.scss';
+
+.j-table {
+  width: 100%;
+  text-align: left;
+
+  tbody {
```

Comment:
if we are al ready referencing al the elements with a custom BEM class maybe lets do the same for the body.